pom.xml

```diff
@@ -111,11 +111,10 @@
     <!-- jfim: for Kafka 0.9.0.0, use zkclient 0.7 -->
     <kafka.version>0.9.0.1</kafka.version>
     <zkclient.version>0.7</zkclient.version>
-    <!-- TODO: bump up the jackson version once we update jersey version to 2.28 or later -->
-    <jackson.version>2.8.3</jackson.version>
+    <jackson.version>2.9.8</jackson.version>
     <async-http-client.version>1.9.21</async-http-client.version>
-    <jersey.version>2.23</jersey.version>
-    <swagger.version>1.5.10</swagger.version>
+    <jersey.version>2.28-RC4</jersey.version>
```

Comment:
`RC` versions are release candidates, not the official releases. If `2.27` version contained the fix that we need, I would use that.

Reply:
Our best bet is with 2.28 now. Hopefully we will get a release soon that we can incorporate.